src/core/base-runner.js

```diff
@@ -62,7 +62,7 @@ export async function runAll(plugs) {
     try {
       await task(respecConfig);
     } catch (err) {
-      console.error(err);
+      pub("error", err);
```

Comment:
I don't think we want to do this, unfortunately: the reason is that we don't want to surface JS errors to end users, as they can't actually fix those. We only `pub("error")` for things users can fix.

Reply:
I still think we should have a way to show that something failed in a fatal way. A silent failure is never helpful for both authors and devs.

Reply:
Maybe something like `pub("fatal", err)`?

Reply:
I'll revert that commit in this PR, we can discuss more in #1760.

Reply:
Related: https://github.com/w3c/respec/issues/1741